src/utils/getProjectMembers.ts

```diff
@@ -1,15 +1,13 @@
-import get from "lodash/get";
-import size from "lodash/size";
 import type { Maybe } from "../types";
 import type { Project, Member } from "../services/space/types";
 
 const getProjectMembers = (project?: Maybe<Project>): Member[] => {
   const members = [
-    ...get(project, ["memberProfiles"], []) || [],
-    ...get(project, ["adminProfiles"], []) || [],
+    ...(Array.isArray(project?.memberProfiles) ? project?.memberProfiles ?? [] : []),
```

Comment:
Shorter: `project?.memberProfiles ?? []`

Reply:
I'm not sure about the response because it comes from the API. It seems to be better to recheck.

Reply:
This could mask a change in their API, if we're following the docs, then the short version will be fine, then if it breaks we are aware of it via logging  (instead of silently not working)

Reply:
fixed